mlflow/utils/databricks_utils.py

```diff
@@ -439,34 +475,65 @@ def get_databricks_host_creds(server_uri=None):
         MlflowHostCreds which includes the hostname and authentication information necessary to
         talk to the Databricks server.
     """
+    # Since we do not record OAuth expiration time in OAuth file, perform periodic refresh
+    # of OAuth cache here. As currently configured (02/24) OAuth token in model serving environment
+    # guaranteed to have at least 30 min remaining on TTL at any point in time but refresh at higher
+    # rate here to be safe and in case those values change in the future.
+    OAUTH_CACHE_REFRESH_DURATION = 10 * 60
```

Comment:
Why do we need this OAUTH_CACHE_REFRESH_DURATION? Is this for refreshing cache after this period has elapsed? 

if so, can you add the units of time? 

OAUTH_CACHE_REFRESH_DURATION_SECONDS / OAUTH_CACHE_REFRESH_DURATION_MINS etc